src/markdown.js

```diff
@@ -11,11 +11,34 @@ ${code}
 </stache-code-block>
 `;
 
-  renderer.heading = (text, level) =>
-`<stache-page-anchor level="${level}">
-${text}
+  renderer.heading = (text, level) => {
+    let parsedHeading = '';
+
+    switch (level) {
+      case 1:
```

Comment:
Just making sure, level comes through as a number, not a string?

Reply:
correct